ding/utils/default_helper.py

```diff
@@ -441,8 +442,16 @@ def split_data_generator(data: dict, split_size: int, shuffle: bool = True) -> d
                 batch[k] = None
             elif k.startswith('prev_state'):
                 batch[k] = [data[k][t] for t in indices[i:i + split_size]]
-            elif isinstance(data[k], list) or isinstance(data[k], tuple):
-                batch[k] = [t[indices[i:i + split_size]] for t in data[k]]
+            # elif isinstance(data[k], list) or isinstance(data[k], tuple):
+            #     batch[k] = [t[indices[i:i + split_size]] for t in data[k]]
+            elif k == 'logit':  # for continuous action
```

Comment:
keep compatible, add new if-else branch in `isinstance(data[k], list) or isinstance(data[k], tuple)`